app/views/lab_results/new.html.erb

```diff
@@ -0,0 +1,7 @@
+<section class="hero is-small is-info">
+  <div class="hero-body">
+    <p class="title">New Lab Result</p>
```

Comment:
Can we add a back button to this page redirecting to the patient show page, similarly to how there is one on the new immunization page?

Reply:
resolved